### ISSUE TITLE
Update to swiper 9/10

### ISSUE THREAD

**Issue by Marius-Romanus** [twinssbc/Ionic2-Calendar#646]
Hello! Swiper has just upgraded to a major version, v9. Thank you very much for keeping this project so interesting and for updating it! ^^ and the new name ionic6-calendar also gives it a fresh look.

https://github.com/nolimits4web/swiper/releases

**Edit:**
I edit because Vladimir Kharlampidi is not stopping and has released version 10, luckily I think there is not as much change as from 8 to 9

https://swiperjs.com/migration-guide-v10

**Comment by twinssbc** [comment]
yeah, starting form v9, Swiper no longer has Angular component, but Web Component instead (https://swiperjs.com/element). Need to do more compatibility testing.

**Comment by Eraldo** [comment]
Does it already work with swiper v9? :D

**Comment by josuelmm** [comment]
For when this integration?

**Comment by sdsoldi** [comment]
Yes please, some info about update? Ionic 7 examples are with swiper 9 https://ionicframework.com/docs/angular/slides

**Comment by Eraldo** [comment]
I upgraded to swiper 9 and this package now broke for me. :|
```
Error: node_modules/ionic2-calendar/calendar.module.d.ts:9:21 - error TS2307: Cannot find module 'swiper/angular' or its corresponding type declarations.
```

**Comment by Gautammer** [comment]
Hello @Eraldo you can use old version of swiperjs to user swiper/angular  `npm i swiper@8.4.7 --f`

**Comment by Eraldo** [comment]
> Hello @Eraldo you can use old version of swiperjs to user swiper/angular `npm i swiper@8.4.7 --f`

Thanks for the reply. Unfortunately I'm already using swiper 9 in my other components. ^^
`npm i swiper@8.4.7 --f` would probably override that, correct?

**Comment by twinssbc** [comment]
I have published the version 2.1.0 which support swiper 10.1.0. Feel free to give a try.
